### DISCUSSION TITLE
慢跑

### DISCUSSION THREAD

**Discussion by gigitilto** [1101-GS4719-C/Python-Community-GS4719-C#3688] [General]
之前我對於我的身體健康狀況有點害怕，恰好同學偶爾會慢跑，多運動也可以幫助入眠，我便跟著開始了。目前我進行了一個多月的慢跑了，感覺體力有所提升，相對的食慾也變好，一開始的睡眠品質確實變得很好，但隨著時間過去，我又逐漸有睡眠困難了，這意思是身體在告訴我以現在的慢跑量我的身體已經不再需要在睡眠時修復破損的肌肉組織嗎？我需要把慢跑距離在往上拉嗎？

**Comment by BOSENCHANG2004** [comment]
前一周開始嘗試開始慢跑，因為覺得自己的身體健康狀況越加的不對勁，我希望今後能夠把慢跑加入我的生活中，讓運動變成我的習慣，在這裡發誓一定要過一個健康的生活。

**Comment by hollowha** [comment]
感覺是要增強運動強度，因為身體已經適應之前的強度了

**Comment by RoyLin1014** [comment]
我覺得慢跑相當不錯，煩心的時候去跑個五圈，可以暫時停止腦袋的聲音，你的狀況我沒有相關經驗感覺可以詢問專業的醫生
